regression/scripts/airfoil_analysis/airfoil_panel_method_test.py

```diff
@@ -62,22 +67,22 @@ def main():
     diff_CL = np.abs(airfoil_properties.Cl[4,1] - xfoil_data_1.Cl) 
     print('\nCL difference')
     print(diff_CL)
-    assert np.abs((airfoil_properties.Cl[4,1]  - xfoil_data_1.Cl)/xfoil_data_1.Cl) < 5e-1
+    assert np.abs((airfoil_properties.Cl[4,1]  - xfoil_data_1.Cl)/xfoil_data_1.Cl) < 3e-1
```

Comment:
I think we talked about this already. It's a matter of time before these lines bite us. This needs to be.

assert (np.abs((airfoil_properties.Cl[4,1] - xfoil_data_1.Cl)/xfoil_data_1.Cl)-EXPECTED ERROR) < 1e-6

Reply:
issue resolved